README.md

```diff
@@ -1,6 +1,6 @@
 # Build@Mercari Training Program 
 
-This is @<your github id>'s build training repository.
+This is @WANGyuquan1025's build training repository.
```

Comment:
I believe your Github ID is `WANGyuquan01`. Can you please update the PR accordingly?